packages/core/src/view/canvas/SvgCanvas2D.ts

```diff
@@ -1819,9 +1820,9 @@ class SvgCanvas2D extends AbstractCanvas2D {
         }
 
         bbox = new Rectangle(
-          (x + 1) * s.scale,
+          x  * s.scale,
           (y + 2) * s.scale,
-          w * s.scale,
+          (w +1) * s.scale,
           (h + 1) * s.scale
         );
```

Comment:
I am pretty sure the previous implementation cames from the orignal mxGraph.
There are a lot of place where there are `+1` on position, width or height. It is very hard to do the review here. I don't have time to dig into the 52 commits to find which example may need the change and how this change impacts other examples.